dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -96,6 +98,42 @@ def test_add_profilers(self, *mocks):
         self.assertEqual('multiple files', merged_profile.encoding)
         self.assertEqual('multiple files', merged_profile.file_type)
 
+    @mock.patch('dataprofiler.profilers.profile_builder.'
+                'ColumnPrimitiveTypeProfileCompiler')
+    @mock.patch('dataprofiler.profilers.profile_builder.'
+                'ColumnStatsProfileCompiler')
+    @mock.patch('dataprofiler.profilers.profile_builder.'
+                'ColumnDataLabelerCompiler')
+    @mock.patch('dataprofiler.profilers.profile_builder.DataLabeler')
+    def test_stream_profilers(self, *mocks):
```

Comment:
this ensures null has row/is row, total samples, duplicate rows are all updated correctly.